app/Services/EventsApiClient.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace App\Services;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\MultipartStream;
+use GuzzleHttp\Exception\GuzzleException;
+
+class EventsApiClient
+{
+    private $client;
+
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
+
+    public function getEvents($keyword)
+    {
+        try {
+            $response = $this->client->request('GET', 'http://localhost:8001/api/events', [
+                'headers' => [
+                    'Accept' => 'application/json',
+                ],
+                'query' => [
+                    'keyword' => $keyword,
+                ],
+            ]);
+
+            return json_decode($response->getBody(), true);
+        } catch (GuzzleException $exception) {
```

Comment:
What is the meaning of this in every function? You are catching and then immediatelly throwing again exception. So who copy who? You copied Kristjan or Kristjan copied you?